src/views/primary-auth/PrimaryAuthForm.js

```diff
@@ -167,8 +173,13 @@ define([
       } else if (!this.settings.get('features.passwordlessAuth')) {
         this.getInputs().toArray()[1].focus();
       }
-    }
+    },
 
+    postRender: function() {
+      if (this.settings.get('features.trackTypingPattern')) {
+        TypingUtil.track('okta-signin-username');
```

Comment:
2 questions here
- is the selector assumed as an id? Asking since we don't pass `#` or `.` or anything specifying what type of selector it is
- do we want to do this in `postRender`? or do we need to start tracking on focus for the username field?